### ISSUE TITLE
Getting rename or repackaging error.

### ISSUE THREAD

**Issue by HikmetTenis** [hibersap/hibersap-sapjco3#1]
Hi

I got jco version 12 and when I try to use it, it gives me "Illegal JCo archive "sapjco3-3.0.jar". It is not
allowed to rename or repackage the original archive "sapjco3.jar" error, I followed the example which is given in your web site, maven adds version to the file and jco doesnt like that, what should I do?

Thanks.

**Comment by jyjsjd** [comment]
I got the same error.

**Comment by fischey** [comment]
One possible solution for this problem was for me to rename the dependency with the maven-dependency-plugin. For the tests I removed the original maven dependency from classpath and added the renamed (now sapjco3.jar) to the classpath again. A better solution did not come in my mind till now.

My actual configuration looks like this:
- the dependency in the pom 

```
<dependency>
<groupId>com.sap</groupId>
<artifactId>jco</artifactId>
<version>3.0.11</version>
<scope>provided</scope>
</dependency>
```

the configuration from the plugins:
[Code-Snippet for the configuration](https://gist.github.com/fischey/01af7776d73946988c275986fd229a4d)

The idea of copying and renaming the artifact is from @cerker himself from the SourceForge-Website(https://sourceforge.net/p/hibersap/discussion/813164/thread/52913577/#37e4). But the system-scope approach did not worked for me. So I combined the approach from hibersap to add the native dependecies to the library.path with the earlier mentioned approach from @cerker.

This solution works for me executing integration tests. For the deployment I use the scope provided for the sapjco.jar.

I hope it helps you.

Best Regards René
